cmd/util/cmd/epoch-recovery/main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "github.com/onflow/flow-go/cmd/util/cmd/epoch-recovery/cmd"
+
+func main() {
+	cmd.Execute()
+}
```

Comment:
Since we are adding a new executable here, please add a corresponding target to the `Makefile`. See [here](https://github.com/onflow/flow-go/blob/d615293d74ecf6438d71bd0a95733ae0d9611f4c/Makefile#L385-L404) for some examples. `docker-all-tools` should build this new exe as well

Reply:
Did you mean you wanted to target to just build the `epoch-recovery` CLI? 

Because currently this should also include the `epoch-recovery`. It should be tied to the `util` command line.

```
PHONY: docker-build-utildocker-build-util:	docker build -f cmd/Dockerfile --ssh default --build-arg TARGET=util --target production \		-t "$(CONTAINER_REGISTRY)/util:latest" -t "$(CONTAINER_REGISTRY)/util:$(SHORT_COMMIT)" -t "$(CONTAINER_REGISTRY)/util:$(IMAGE_TAG)" .
PHONY: tool-utiltool-util: docker-build-util	docker container create --name util $(CONTAINER_REGISTRY)/util:latest;docker container cp util:/bin/app ./util;docker container rm util<br class="Apple-interchange-newline">
```

Reply:
👍  This works too. If it's part of the `util` target, no need to add a separate target.